roles/openshift_web_console/files/console-template.yaml

```diff
@@ -37,7 +37,10 @@ objects:
   spec:
     replicas: "${{REPLICA_COUNT}}"
     strategy:
-      type: Recreate
+      type: RollingUpdate
+      rollingUpdate:
+        # behave like a recreate deployment, but don't wait for pods to terminate
```

Comment:
Even with this setup you can have different versions running in parallel because maxSurge defaults to 25% and the deployment controller always scales up first if it can. Why is it problematic to run different versions in parallel?

Reply:
Mismatched JS can cause errors because we don't have sticky sessions. (This is a non-issue in converged console next release since we have a route instead of a master proxy.)